anomalydetection/features.py

```diff
@@ -20,3 +20,12 @@ def lag_time_series(time_series: pd.Series, lags):
         lagged_time_series[str(lag)] = time_series.shift(lag)
 
     return pd.concat(lagged_time_series, axis=1)
+
+
+def create_dataset(X, y, time_steps=1):
+    Xs, ys = [], []
+    for i in range(len(X) - time_steps):
+        v = X.iloc[i:(i + time_steps)].values
```

Comment:
What happens if input data is not in the assumed order, i.e. if timestamps are either in random order (unlikely) or descending vs ascending? If I'm not missing something obvious and this comment is indeed relevant, I think we have some code in another project we could use that handles this. Let me know :)

Reply:
Yep that is an input assumption. Please feel free to solve this or I can simply make a comment

Reply:
Would be great if you could make a comment, I'll try to find time at a later point to substitute with the other code.